### PR TITLE
ZH-trans: update encrypt-data.md

### PR DESCRIPTION
**Pull request by xichengliudui**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Please delete this note before submitting the pull request.
>
> For 1.14 Features: set Milestone to 1.14 and Base Branch to dev-1.14
> 
> For Chinese localization, base branch to release-1.12
>
> For Korean Localization: set Base Branch to dev-1.13-ko.<latest team milestone>
>
> Help editing and submitting pull requests:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> Help choosing which branch to use:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/encrypt-data.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
-<!--
 ---
 reviewers:
 - smarterclayton
-title: Encrypting Secret Data at Rest
+title: 加密静态 Secret 数据
```
Comment (chenrui333) [inline content/zh/docs/tasks/administer-cluster/encrypt-data.md]:
```suggestion
title: 静态加密 Secret 数据
```

**Comment by xichengliudui** [comment]
已修改。

**Pull request merged**